apps/small_filters/conv3x3a32.cpp

```diff
@@ -0,0 +1,61 @@
+#include "Halide.h"
+
+using namespace Halide;
+using namespace Halide::Internal;
+IRPrinter irp(std::cerr);
+
+class Conv3x3a32 : public Generator<Conv3x3a32> {
```

Comment:
This Generator is almost identical to Conv3x3a16; they could trivially be combined into a single Generator by adding 

    GeneratorParam<Type> intermediate_type{...};

and specializing a couple of lines based on that.